api-streaming/constants.py

```diff
@@ -0,0 +1,10 @@
+PLUGIN_NAME = "UFM_API_Streaming"
```

Comment:
why this file needed? maybe redundant and put all this code into main file so it will be only one